.gitignore

```diff
@@ -45,3 +45,5 @@ autowiring.kdev4
 autowiring.VC.db
 .vscode
 AutowiringVersion.h
+
+*.VC.db*
```

Comment:
I see now. I have `Browse.VC.db` and `Solution.VC.db{,-shm,-wal}`.

Should remove the now-redundant `autowiring.VC.db` above then?